go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/equinor/radix-api
 
-go 1.20
+go 1.21
 
 require (
-	github.com/equinor/radix-common v1.4.0
+	github.com/equinor/radix-common v1.5.0
 	github.com/equinor/radix-job-scheduler v1.8.2
 	github.com/equinor/radix-operator v1.40.5
```

Comment:
Update `github.com/equinor/radix-job-scheduler` and `github.com/equinor/radix-operator` to newest version

Reply:
Fixed :)